syslog-ng-helm-chart/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*.orig
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+.vscode/
```

Comment:
Can you remove the irrelevant files from here?

Reply:
this is a helm best practice, helm create thats for you when you create a new chart. Its based on the community experience. This is not relative to my workspace. But let me know if you still want to get rid of this. makes no difference ;)